content/en/docs/v3.6/learning/glossary.md

```diff
@@ -88,6 +88,11 @@ A point-in-time backup of the etcd cluster state.
 
 The physical storage backing the cluster keyspace.
 
+## Term
+
+A term is a monotonically increasing integer that is associated with each leader election in the Raft algorithm.
```

Comment:
Would be good to mention that for "For a term there can only be one elected leader and term is incremented on leader change". 

I think this is accurate +/- pre-vote and I think that leader election can fail so there might not be a leader for a term.

Reply:
@serathius  I added your suggestion and removed the second sentence. Does it make sense now?